newton/_src/geometry/types.py

```
@@ -405,3 +405,135 @@ def __hash__(self) -> int:
+        self._data = np.array(data, dtype=np.float32).reshape(nrow, ncol)
+        self.nrow = nrow
+        self.ncol = ncol
+        self.size = size  # (size_x, size_y, size_z, size_base)
+        self.is_solid = is_solid
+        self.has_inertia = compute_inertia
+        self.warp_array = None  # Will be set by finalize()
+        self._cached_hash = None
+
+        if compute_inertia:
+            # Approximate heightfield inertia as a box with average height
+            from .inertia import compute_box_inertia  # noqa: PLC0415
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)